documentation/src/main/asciidoc/user_guide/clustered_lock.adoc

```diff
@@ -0,0 +1,200 @@
+== Clustered Lock
+
+Clustered lock is a new building block introduced in Infinispan 9.2
+It is a lock distributed and shared among all nodes in the Infinispan cluster and currently provides a way to execute
```

Comment:
"It is a lock which is distributed and shared..."

Reply:
ok